application/models/organization.go

```diff
@@ -76,28 +62,98 @@ func (o *Organization) GetAuthProvider() (auth.Provider, error) {
 	return &auth.EmptyProvider{}, fmt.Errorf("unsupported auth provider type: %s", o.AuthType)
 }
 
-func FindOrgByUUID(uuid string) (Organization, error) {
+func (o *Organization) FindByUUID(uuid string) error {
 
 	if uuid == "" {
-		return Organization{}, fmt.Errorf("error: access token must not be blank")
+		return fmt.Errorf("error: access token must not be blank")
 	}
 
-	org := Organization{}
+	if err := DB.Where("uuid = ?", uuid).First(o); err != nil {
+		return fmt.Errorf("error finding org by uuid: %s", err.Error())
+	}
 
-	queryString := fmt.Sprintf("uuid = '%s'", uuid)
+	return nil
+}
 
-	if err := DB.Where(queryString).First(&org); err != nil {
-		return Organization{}, fmt.Errorf("error finding org by uuid: %s", err.Error())
+func (o *Organization) FindByDomain(domain string) error {
```

Comment:
Would this work?
```go
DB.Eager().Where("domain = ?", domain).First(&orgDomain)
...
*o = orgDomain.Organization
```

After reading below, I see that's the way it was...you must have changed it for a reason.